### DISCUSSION TITLE
Building Informations / Opening Times / Guide

### DISCUSSION THREAD

**Discussion by ghost** [astarub/campus_app#70] [Ideas]
I think it would be generally nice to Include a Section Opening Times in the Style of or instead of the Guide. Currently i dont think the Guide is very useful but having information to specific locations would be nice. I imagine that the guide lists places like UB or Kukaf or Mensa instead and you find Location (open in default Map), Opening Times, Website and a short description. Imo this would be useful to have (maybe even sort by open) and better than the current guide or atleast a nice addition. I dont think the current Guide is very useful.

**Comment by C4s4r** [comment]
The current state of the guide page is meant to change in the near future, as future features should be added there.
When this happens, the guide itself will be a button "FAQ" or "Campus FAQ" that leads to a subpage, where the current entries are displayed.

Your ideas regarding an overview for each of the places is good and could be one of the future contents on this page.

As for now, it's planned to display a wallet and a _Campus Map_ on this page.
